packages/frontend/src/main/components/viewer/ViewerFilters.vue

```diff
@@ -201,8 +201,10 @@ export default {
         filter.data = rawFilter
         const key = rawFilter.key
         // Handle revit params (a wee bit of FML moment)
-        if (key.startsWith('parameters.')) {
+        if (key.startsWith('parameters.') || key.startsWith('properties.')) {
           if (key.endsWith('.value')) {
+            console.log('pushing', key)
+            console.log('pushing', key)
```

Comment:
Did you mean to log this twice?

Reply:
i probably meant to not log this at all! i'll fix!